RobotCode/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -24,7 +38,11 @@ public RobotContainer()
 
 	private void configurePrimaryBindings()
 	{
-
+		// Testing for Shooter Mount code. Remove before merging into dev.
```

Comment:
Move this stuff into `configureSecondaryBindings`. It's secondary controller stuff, rather than primary controller stuff.